landing_page/templates/landing_page/about.html

```diff
@@ -19,6 +20,33 @@ <h2>What is ReeldIn?</h2>
       provide the best recommendations possible. We're here to make your
       movie-watching experience as enjoyable as possible.
     </p>
+
+    <br />
+    <h2>FAQ</h2>
+    <br />
+    <h3>Q: How do I make an account?<br /></h3>
+    <br />
+    <p>
+      A: To make an account, click the "Login" button in the top right corner of
+      the page. You'll be prompted to input your email and password. Once you've
+      done that, you'll have access to all of ReeldIn's features.
+    </p>
+    <br />
+    <h3>Q: How do I get recommendations? <br /></h3>
+    <br />
+    <p>
+      A: To get recommendations, click the "Get Recommendations" button on the
+      home page. You'll be prompted to input your favorite movies, and ReeldIn
+      will provide you with a list of similar movies.
+    </p>
+    <br />
+    <h3>Q: How does ReeldIn work? <br /></h3>
+    <br />
+    <p>
+      A: ReeldIn uses a combination of user input and machine learning to
```

Comment:
need to revisit this as i don't think we have any machine learning planned?
at least i don't...

Reply:
The filtering I'll be doing for the recommendation system for imported user data uses Machine learning algorithms